src/ngd.ts

```diff
@@ -0,0 +1,173 @@
+// src/handlers/ngd.ts
+
+import { requestNGD as request, get } from "./utils/request.js";
+import { Feature, type FeatureCollection } from "geojson";
+import { buildNGDUrl } from "./utils/url.js";
+import { validateParams } from "./utils/validate.js";
+import { BBox, Config } from "./types.js";
+import { initialiseConfig } from "./utils/config.js";
+import fetch from "node-fetch";
+
+export { ngd };
+
+// Types
+interface NGDExtent {
+  spatial: {
+    bbox: number[][];
+    crs: string;
+  };
+  temporal: {
+    interval: string[][];
+    trs: string;
+  };
+}
+
+interface NGDLink {
+  href: string;
+  rel: string;
+  type: string;
+  title: string;
+}
+
+interface NGDCollection {
```

Comment:
should ngdcollection extend geojson?

Reply:
Don't think it's related to geojson - the collections are just standard json, listing the collection attributes